.ado/azure-pipelines.yml

```diff
@@ -7,69 +7,84 @@ pool:
 variables:
   CI: true
 
-steps:
-  - checkout: self
-    persistCredentials: true
+jobs:
+  - job: ApplePR
+    displayName: Apple PR
+    pool:
+      vmImage: 'macos-10.15'
+      demands: ['xcode', 'sh', 'npm']
+    timeoutInMinutes: 60 # how long to run the job before automatically cancelling
+    cancelTimeoutInMinutes: 5 # how much time to give 'run always even if cancelled tasks' before killing them
+    strategy:
+      maxParallel: 4
+      matrix:
+        ios:
+          relative_directory: 'ios'
+          xcode_sdx: 'ios'
+          xcode_configuration: 'Debug'
+          xcode_workspacePath: 'apps/apple/FluentUITester/ios/FluentUITester.xcworkspace'
+          xcode_scheme: 'FluentUITester' #todo add debug/release schemes here
+          xcode_actions: 'build'
+          xcode_useXcpretty: true
+          xcode_destination: 'platform=iOS Simulator,OS=latest,name=iPhone 8'
```

Comment:
Why iPhone 8?

Reply:
It's what react native tests against and I wanted to be consistent

Reply:
Is there actually any advantage to consistency in this case? We're not testing react-native and probably want to intentionally choose the device we think is best to test fluentui-react-native?